cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -126,7 +128,7 @@
     <!-- ros_control plugin -->
     <gazebo>
       <plugin name="ros_control" filename="libhwi_switch_gazebo_ros_control.so">
-        <robotNamespace>${name}</robotNamespace>
+        <robotNamespace>$(arg robot_namespace)/${name}</robotNamespace>
```

Comment:
the default (`""`) changes the current behavior
i.e. it adds a leading `/`
I don't know the consequences, but I'd rather like to see that it defaults to current behavior **without any change**

this review comment applies for all base.urdf.xacro files

Reply:
foreseeing bad implications without having tested every possible usecase (mojin usecases) is tricky to tell

while not ideal, your alternative is less intrusive for everyone not using multirobot setup